### ISSUE TITLE
Duplicating layers does not generate a new layer event

### ISSUE THREAD

**Issue by iwehrman** [adobe-photoshop/generator-core#307]
As reported here: https://groups.google.com/forum/#!topic/adobe-generator-dev/5wF-Ro0KMBs

---
If you duplicate a layer (right click->Duplicate layer...) there is no event for the new layer, just a "changed=true" for whole document and selection of new layer
```javascript
  "version":"1.2.0",
  "timeStamp":1433319803.263,
  "count":3,
  "id":1214,
  "changed":true,
  "selection":[1]
  }
```
Steps to replicate (Photoshop CC 2014 latest version)

1. Create a new document
2. Select layer
3. Right click on the layer and select "Duplicate layer"

**Comment by jaredadobe** [comment]
This needs to be fixed in photoshop-core. I have an open task on my backlog and the code written to fix it. Just need to get reviewed. It will land in the next release of photoshop.

**Comment by joelrbrandt** [comment]
User reports this issue persists in CC 2015.

**Comment by joelrbrandt** [comment]
Thanks for taking this on, @jaredadobe !

**Comment by joelrbrandt** [comment]
Filed Watson bug 4036322

**Comment by timothynoel** [comment]
This should be fixed in Photoshop.next.  We'll send out layer:added instead of  image:changed:true.  Commands affected are group, duplicate, dragging layers between documents.  You may also see a clip: change if clipping had to be changed on an existing layer to accommodate the add.  Also fixed the smart object message to send layer:smartObject{}, instead of layer:changed:true

(image:changed:true or layer:changed:true, just means the command or operation hasn't been retrofitted for change messages, so consider those mean everything changed in the image/layer).

**Comment by joelrbrandt** [comment]
FYI, user indeed reports this is fixed in 2015.1.1. Thanks, @timothynoel
